src/components/CodeInput/inputItems.js

```diff
@@ -20,11 +21,19 @@ const InputItems = React.forwardRef((props, ref) => {
         id,
     } = props;
 
+    const handleOnFocus = e => {
+        if (readOnly || disabled) {
+            ref.current.blur();
```

Comment:
check this, because I think that when you pass disabled or readonly to the input itself then it does not get the focus so you don't need to call blur programatically

Reply:
I believe that when I did this test my used example was wrong, that make me think I should disable programmatically... this is misterious hahaha...

Now I confirmed it is not needed